src/main/resources/com/arcbees/website/client/resources/css/pages/home.gss

```diff
@@ -9,7 +9,7 @@
 /* -- Page : home ---------------------------------------- */
 
 .home .header:before {
-    content: "\e601";
+    content: "\e630";
```

Comment:
There's probably a better way of doing it but I wanted to do a quick fix because the visual was kinda messed up